sdk/containerregistry/azure-containerregistry/azure/containerregistry/aio/_async_base_client.py

```diff
@@ -32,8 +31,7 @@ class ContainerRegistryBaseClient(object):
 
     def __init__(
         self, endpoint: str, credential: "AsyncTokenCredential", **kwargs
-    ):  # pylint:disable=client-method-missing-type-annotations
-        # type: (str, TokenCredential, Dict[str, Any]) -> None
```

Comment:
just curious, but why did you remove this line?

Reply:
This is async, so always Python3, we can add the type hints directly into the signature.